controllers/expose/ingressClassCreator.go

```diff
@@ -20,29 +14,31 @@ import (
 
 const (
 	ingressClassCreationEventReason = "IngressClassCreation"
+	serviceDiscoveryDeployName      = "k8s-service-discovery-controller-manager"
 )
 
-var k8sCesLabels = map[string]string{"app": "ces", "app.kubernetes.io/name": "k8s-service-discovery"}
-
-type eventRecorder interface {
-	record.EventRecorder
-}
-
 // ingressClassCreator is responsible to create a cluster wide ingress class in the cluster.
 type ingressClassCreator struct {
-	client        client.Client
-	className     string
-	namespace     string
-	eventRecorder eventRecorder
+	className             string
+	namespace             string
```

Comment:
The namespace of the structs is only used as a reference in a comment. Can the namespace be removed?